src/middlewared/middlewared/plugins/kubernetes_linux/lifecycle.py

```diff
@@ -116,9 +112,6 @@ def status_change(self):
     async def status_change_internal(self):
         await self.validate_k8s_fs_setup()
         await self.middleware.call('service.start', 'docker')
```

Comment:
What ensures that docker daemon is properly initialized now?

Reply:
It seems the latest update of our mirror has fixed that with updated packages. I ran some tests and this call only returns when the daemon has properly initialised itself.